packages/p2p/src/components/introducing-floating-rates-modal/introducing-floating-rates-modal.scss

```diff
@@ -0,0 +1,21 @@
+.introducing-floating-rates-modal {
+    &__image {
+        background-size: contain;
+        background-repeat: no-repeat;
+        background-position: center;
+        width: 100%;
+        margin-bottom: 2rem;
+    }
+
+    &__checkbox {
+        margin-top: 1.2rem;
+
+        .dc-checkbox__box {
+            margin-left: unset !important;
```

Comment:
can we avoid using `!important`?

Reply:
Removed it now, previously I had to use `important` since it wasn't working